presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkConfiguration.java

```diff
@@ -23,10 +23,13 @@
 
 public class PrestoSparkConfiguration
 {
+    public static final String METADATA_STORAGE_TYPE_KEY = "metadata_storage_type";
+    public static final String METADATA_STORAGE_TYPE_LOCAL = "LOCAL";
+
     private final Map<String, String> configProperties;
     private final String pluginsDirectoryPath;
     private final Map<String, Map<String, String>> catalogProperties;
-    private final String metadataStorageType;
+    private final Map<String, String> prestoSparkProperties;
```

Comment:
Would you only need String types of properties? I think we can create something more generic for passing properties?

Reply:
Most of the time properties are type of `Map<String, String>`, I feel it should be sufficient. Also, I am following examples of other properties here in `PrestoSparkConfiguration`. What do you suggest?

Reply:
More generic way, like OrcReadOptions can be used.